core/src/main/scala/kafka/server/BrokerMetadataCheckpoint.scala

```diff
@@ -94,11 +103,16 @@ class RawMetaProperties(val props: Properties = new Properties()) {
 }
 
 object MetaProperties {
+  def apply(clusterId: String, nodeId: Int): MetaProperties = {
+    MetaProperties(clusterId, nodeId, MetadataVersions.latest().version())
```

Comment:
If a user is using the format tool to initialize a cluster, we default to the "latest" metadata.version